storage/simulation/cluster.go

```diff
@@ -283,7 +283,7 @@ func (c *Cluster) prepareActions() {
 				rep.rebalance = r.allocator.ShouldRebalance(storeID)
 				// Set the priority to 1 so that rebalances will occur in
 				// performActions.
-				rep.priority = 1
+				rep.priority = 1 + c.rand.Float64()
```

Comment:
One of the ideas behind the simulator was to avoid introducing randomness wherever possible (except for random splits, but I'm thinking of even removing those).  I see your what you mean on why no rebalancing would occur, but perhaps the correct place to do this is in `performActions` instead of changing the priorities?